app/uk/gov/hmrc/digitalcontactstub/controllers/PayAsYouEarnController.scala

```diff
@@ -0,0 +1,65 @@
+/*
+ * Copyright 2023 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package uk.gov.hmrc.digitalcontactstub.controllers
+
+import play.api.Logging
+import play.api.libs.json.JsValue
+import play.api.mvc.{Action, MessagesControllerComponents}
+import uk.gov.hmrc.digitalcontactstub.models.paye.PayeOutput
+import uk.gov.hmrc.play.bootstrap.frontend.controller.FrontendController
+
+import javax.inject.{Inject, Singleton}
+import scala.concurrent.Future
+
+
+@Singleton
+class PayAsYouEarnController  @Inject
+(
+  cc: MessagesControllerComponents
+)
+  extends FrontendController(cc)
```

Comment:
Does it need to be a frontend controller?

Reply:
Good question, no it does not. I don’t  know how this works, if the project is a frontend project as defined in the application.conf file, can you still add a controller as a backend?

Reply:
Yes